package.json

```diff
@@ -11,13 +11,14 @@
 	"author": "wrangler@cloudflare.com",
 	"workspaces": [
 		"packages/*",
-		"fixtures/*"
+		"fixtures/*",
+		"packages/templates/packages/create-cloudflare"
 	],
 	"scripts": {
 		"build": "npm run build --workspace=wrangler --workspace=jest-environment-wrangler --workspace=pages-plugin-example --workspace=wranglerjs-compat-webpack-plugin",
 		"check": "run-p check:* --aggregate-output --continue-on-error",
 		"check:format": "prettier . --check --ignore-unknown",
-		"check:lint": "eslint \"packages/**/*.[tj]s?(x)\" --cache --cache-strategy content --max-warnings=0",
+		"check:lint": "eslint \"packages/**/*.[tj]s?(x)\" --cache --cache-strategy content --max-warnings=0 --config ./.eslintrc.js",
```

Comment:
ESlint CLI was not using the config properly, explicitly defining the config to use in ESLint CLI [2c28cd2](https://github.com/cloudflare/wrangler2/pull/2498/commits/2c28cd24fa4b1e7aba267052c204f308ea94029f)